src/Scanner.js

```diff
@@ -107,7 +107,7 @@ class DefaultScanner {
         window.performance?.mark?.('scanner:init:start')
         this.findEligibleInputs(document)
         window.performance?.mark?.('scanner:init:end')
-        this.mutObs.observe(document.body, { childList: true, subtree: true })
+        this.mutObs.observe(document.documentElement, { childList: true, subtree: true })
```

Comment:
By observing the whole html we ensure the script is updated when navigating through pages in SPAs using web workers. This fixes [Removaly](https://app.asana.com/0/1200930669568058/1202313905254477/f).

Reply:
@GioSensation Mostly for my own knowledge, how does this solve the issue? This will mean we'll catch changes in the head, but won't any changes that we care about update the body element too?

Reply:
Some pages swap the `body`, so if you observe changes to the body you basically lose all observing capability. This fixes it.